### PR TITLE
AU Variance Statement preamble changes

### PR DESCRIPTION
**Pull request by JayMurdoch**
Fix for [FHIR-47179](https://jira.hl7.org/browse/FHIR-47179)

Added required text to AU Variance Statement.

### CODE REVIEW

File: input/pagecontent/variance.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
-AU Realm FHIR implementation guide projects are required to follow specific publishing guidelines:
+This version of the AU Core FHIR IG complies with Version 1.0 of the [HL7 Australia FHIR Work Group AU Variance Requirements](https://confluence.hl7.org/display/HA/Process%3A++AU+FHIR+IG+Variance+Requirements?preview=/239212390/281217432/HL7%20AU%20Variance%20Requirements%20v1.0.pdf).
```
Comment (dbojicic) [inline input/pagecontent/variance.md:1]: Missing the change to specify the exact version of the Variance Requirements doc in the hyperlink text.

File: input/pagecontent/variance.md [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
 
-The Variance Statement identifies where an AU Realm FHIR implementation guide does not meet the expectations set by AU Base and/or AU Core. It also identifies resources not profiled in AU Base or AU Core, facilitating the FHIR Work Group's assessment for potential further development. Additionally, the Variance Statement page indicates where the AU Realm FHIR IG fully complies with AU Base or AU Core.
+The Variance Statement identifies where an HL7 Australia published FHIR implementation guide does not meet the expectations set by AU Base and/or AU Core. It also identifies resources not profiled in AU Base or AU Core, facilitating the FHIR Work Group's assessment for potential further development. Additionally, the Variance Statement page indicates where the HL7 Australia FHIR IG fully complies with AU Base or AU Core.
 
 The Variance Statement undergoes review and assessment by the FHIR Work Group as part of the FHIR IG balloting process. For more details on the requirements of a Variance Statement, refer to [AU FHIR IG Variance Requirements](https://confluence.hl7.org/display/HA/Process%3A++AU+FHIR+IG+Variance+Requirements).
```
Comment (dbojicic) [inline input/pagecontent/variance.md:11]: This points to the same document - I think we should update the hyperlink text to match the first paragraph.

**Review by dbojicic — CHANGES_REQUESTED** [review]
One minor comment

File: input/pagecontent/variance.md (hunk 1 above)
Comment (JayMurdoch) [reply input/pagecontent/variance.md:1]: Version specified in text and link is to v1.0 pdf document. Please let me know if I have misunderstood your comment.

File: input/pagecontent/variance.md (hunk 2 above)
Comment (JayMurdoch) [reply input/pagecontent/variance.md:11]: Amended, thanks.

File: input/pagecontent/variance.md (hunk 1 above)
Comment (JayMurdoch) [reply input/pagecontent/variance.md:1]: As discussed in IG triage - hyperlinked 'Version 1.0 of the HL7 Australia FHIR Work Group AU Variance Requirements' to satisfy ballot resolution.

**Review by dbojicic — APPROVED** [review]

**Review by brettesler-ext — APPROVED** [review]
approved with updated PDF document location just made available

**Pull request merged**